src/frontend/src/builder/components/BuilderPriceCounter.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="content__result">
+    <p>Итого: {{ price }} ₽</p>
+    <button
+      type="button"
+      class="button"
+      :disabled="buttonCookDisabled"
+      @click="onCook()"
+    >
+      Готовьте!
+    </button>
+  </div>
+</template>
+
+<script>
+import pizza from "@/static/pizza.json";
+import doughStatuses from "@/common/enums/doughStatuses";
+export default {
+  name: "BuilderPriceCounter",
+  props: {
+    recipe: {
+      type: Object,
+      required: true,
+    },
+    pizzaName: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      pizza,
+      doughStatuses,
+      pizzaOrder: { name: null, recipe: null, price: 0 },
+    };
+  },
+  methods: {
+    getDoughPrice() {
+      var doughPrice = 0;
+      doughPrice = pizza.dough.find(
+        (el) => el.id === this.recipe.doughId
+      ).price;
+      return doughPrice;
+    },
+    getSaucePrice() {
+      var saucePrice = 0;
+      saucePrice = pizza.sauces.find(
+        (el) => el.id === this.recipe.sauceId
+      ).price;
+      return saucePrice;
+    },
+    getSizeCoeff() {
+      var multiplier = 0;
+
+      multiplier = pizza.sizes.find(
+        (el) => el.id === this.recipe.sizeId
+      ).multiplier;
+
+      return multiplier;
+    },
+    calcPriceByIngredient(ingredientId) {
+      var ingredientPrice = 0;
+      ingredientPrice = pizza.ingredients.find(
+        (el) => el.id === ingredientId
+      ).price;
+      return ingredientPrice;
+    },
+
+    getIngredientsPrice() {
+      var ingredientPrice = 0;
+      for (var i = 0; i < this.recipe.ingredients.length; i = i + 1) {
+        ingredientPrice =
+          ingredientPrice +
+          this.calcPriceByIngredient(this.recipe.ingredients[i].ingredientId) *
+            this.recipe.ingredients[i].itemCount;
+      }
+      return ingredientPrice;
+    },
+    onCook() {
+      if (this.pizzaName == null) {
+        alert("Название пиццы обязательно для заполнения");
+      } else {
+        this.pizzaOrder.name = this.pizzaName;
+        this.pizzaOrder.recipe = this.recipe;
+        this.pizzaOrder.price = this.price;
+        this.$emit("onCook", this.pizzaOrder);
+      }
+    },
+  },
+  computed: {
+    price: function () {
+      return (
+        //мультипликатор размера х (стоимость теста + соус + ингредиенты).
+        this.getSizeCoeff() *
+        (this.getDoughPrice() +
+          this.getSaucePrice() +
+          this.getIngredientsPrice())
+      );
+    },
+    buttonCookDisabled: function () {
+      if (
+        this.getIngredientsPrice() === 0 ||
```

Comment:
`return this.getIngredientsPrice() === 0 || this.pizzaName === null || this.pizzaName === ""`